runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerLog.java

```diff
@@ -102,8 +146,52 @@
 			itemContainer.add(slotContainer);
 		}
 
-		add(logTitle, BorderLayout.NORTH);
-		add(itemContainer, BorderLayout.CENTER);
+		itemContainer.repaint();
+	}
+
+	private List<LootTrackerItemEntry> getCombinedItems()
```

Comment:
Remove this method and just move it to rebuildItems() and calculate total in same loop

Reply:
But rebuild items is only executed when the config changes, not everytime a new item/log is added, so the price would always be oudated until the config changed